ql/src/test/results/clientpositive/llap/subquery_in.q.out

```diff
@@ -2818,7 +2818,7 @@ STAGE PLANS:
             Reduce Operator Tree:
               Merge Join Operator
                 condition map:
-                     Left Semi Join 0 to 1
+                     Inner Join 0 to 1
```

Comment:
This plan seems to be semantically different. Could you confirm whether the new one is correct (or maybe I am wrong and they are equivalent)?